src/utils/index.js

```diff
@@ -6,8 +6,24 @@ export const formatRuntime = (runtime) => {
     return `${hours}h ${minutes}m`
 }
 
-const makeApiCall = async (url) => {
-    const response = await fetch(url)
+const makeApiCall = async (url, method, data) => {
+    const response = await fetch(
+        url,
+        {
+            method: method,
+            ...(data && {
+                headers: {
+                    "Content-Type": "application/json",
+                },
+                body: JSON.stringify(data)
+            })
+        }
+    )
+
+    if (response.status === 204) {

```

Comment:
I think you need error handling here. e.g if (res.status !== 200) throw new Error()

Reply:
This if clause is specifical done for updateMovies call which returns 204 status, thus I just need to return without throwing an error